### ISSUE TITLE
Environment conda create warnings?

### ISSUE THREAD

**Issue by Sakura1a2a3a** [huaishengzhu/3MDiffusion#4]
Hi, I am just wondering if the following error is expected. I followed your instruction and used the environment.yml file to install the conda environment. 

Pip subprocess error:
ERROR: Ignored the following versions that require a different python version: 0.0.1 Requires-Python >=3.9,<4.0; 0.12.0.dev0 Requires-Python >=3.9; 0.14.0 Requires-Python >=3.9.0; 0.21.0 Requires-Python >=3.9; 0.22.0 Requires-Python >=3.9; 0.22.0rc1 Requires-Python >=3.9; 0.22.1 Requires-Python >=3.9; 0.22.3 Requires-Python >=3.9; 0.23.0 Requires-Python >=3.10; 0.23.0rc0 Requires-Python >=3.10; 0.23.0rc2 Requires-Python >=3.10; 0.23.1 Requires-Python >=3.10; 0.23.2 Requires-Python >=3.10; 0.23.2rc1 Requires-Python >=3.10; 0.24.0 Requires-Python >=3.9; 0.24.0rc1 Requires-Python >=3.9; 0.25.0 Requires-Python >=3.10; 0.25.0rc0 Requires-Python >=3.10; 0.25.0rc1 Requires-Python >=3.10; 0.25.0rc2 Requires-Python >=3.10; 0.55.2 Requires-Python <3.5; 1.1.0 Requires-Python <3.14,>=3.9; 1.1.0 Requires-Python >=3.9.0; 1.1.0a0 Requires-Python <3.14,>=3.9; 1.1.1 Requires-Python >=3.9.0; 1.2.0 Requires-Python >=3.9; 1.2.0 Requires-Python >=3.9.0; 1.2.1 Requires-Python >=3.9; 1.2.1 Requires-Python >=3.9.0; 1.2.1rc1 Requires-Python >=3.9; 1.3.0 Requires-Python >=3.9; 1.3.1 Requires-Python >=3.10; 1.3.2 Requires-Python >=3.9; 1.4.8 Requires-Python >=3.10; 1.40.2 Requires-Python !=3.9.7,>=3.9; 1.41.0 Requires-Python !=3.9.7,>=3.9; 1.41.1 Requires-Python !=3.9.7,>=3.9; 1.5.0 Requires-Python >=3.9; 1.6.0 Requires-Python >=3.9; 1.6.0rc1 Requires-Python >=3.9; 1.7.0 Requires-Python >=3.10; 1.8.0 Requires-Python >=3.10; 1.9.0 Requires-Python >=3.9; 12.0.0 Requires-Python >=3.9; 18.0.0 Requires-Python >=3.9; 18.1.0 Requires-Python >=3.9; 2.0.8 Requires-Python >=3.9; 2.36.0 Requires-Python >=3.9; 2.36.1 Requires-Python >=3.9; 2.5.0 Requires-Python <3.14,>=3.9; 2.5.0 Requires-Python >=3.9; 2.5.0.post0 Requires-Python >=3.9; 2.5.0rc0 Requires-Python >=3.9; 2.6.2 Requires-Python >=3.9; 2.6.3 Requires-Python >=3.9; 2023.12.9 Requires-Python >=3.9; 2023.7.18 Requires-Python >=3.9; 2023.8.12 Requires-Python >=3.9; 2023.8.25 Requires-Python >=3.9; 2023.8.30 Requires-Python >=3.9; 2023.9.18 Requires-Python >=3.9; 2023.9.26 Requires-Python >=3.9; 2024.1.30 Requires-Python >=3.9; 2024.10.1 Requires-Python >=3.9; 2024.12.12 Requires-Python >=3.10; 2024.2.12 Requires-Python >=3.9; 2024.4.18 Requires-Python >=3.9; 2024.4.24 Requires-Python >=3.9; 2024.5.10 Requires-Python >=3.9; 2024.5.22 Requires-Python >=3.9; 2024.5.3 Requires-Python >=3.9; 2024.6.18 Requires-Python >=3.9; 2024.7.2 Requires-Python >=3.9; 2024.7.21 Requires-Python >=3.9; 2024.7.24 Requires-Python >=3.9; 2024.8.10 Requires-Python >=3.9; 2024.8.24 Requires-Python >=3.9; 2024.8.28 Requires-Python >=3.9; 2024.8.30 Requires-Python >=3.9; 2024.9.20 Requires-Python >=3.10; 3.10.0 Requires-Python >=3.10; 3.10.0rc1 Requires-Python >=3.10; 3.11.0 Requires-Python >=3.9; 3.11.0b0 Requires-Python >=3.9; 3.11.0b1 Requires-Python >=3.9; 3.11.0b2 Requires-Python >=3.9; 3.11.0b3 Requires-Python >=3.9; 3.11.0b4 Requires-Python >=3.9; 3.11.0b5 Requires-Python >=3.9; 3.11.0rc0 Requires-Python >=3.9; 3.11.0rc1 Requires-Python >=3.9; 3.11.0rc2 Requires-Python >=3.9; 3.11.1 Requires-Python >=3.9; 3.11.10 Requires-Python >=3.9; 3.11.11 Requires-Python >=3.9; 3.11.2 Requires-Python >=3.9; 3.11.3 Requires-Python >=3.9; 3.11.4 Requires-Python >=3.9; 3.11.5 Requires-Python >=3.9; 3.11.6 Requires-Python >=3.9; 3.11.7 Requires-Python >=3.9; 3.11.8 Requires-Python >=3.9; 3.11.9 Requires-Python >=3.9; 3.2.0 Requires-Python >=3.9.0; 3.5.0 Requires-Python >=3.9; 3.6.0 Requires-Python >=3.9; 3.6.1 Requires-Python >=3.9; 3.6.2 Requires-Python >=3.9; 3.7.0 Requires-Python >=3.9; 3.7.1 Requires-Python >=3.9; 3.8.0 Requires-Python >=3.9; 3.8.0rc1 Requires-Python >=3.9; 3.8.1 Requires-Python >=3.9; 3.8.2 Requires-Python >=3.9; 3.8.3 Requires-Python <3.13,>=3.9; 3.8.3 Requires-Python >=3.9; 3.8.4 Requires-Python >=3.9; 3.9.0 Requires-Python >=3.9; 3.9.0rc2 Requires-Python >=3.9; 3.9.1 Requires-Python >=3.9; 3.9.1.post1 Requires-Python >=3.9; 3.9.2 Requires-Python >=3.9; 3.9.3 Requires-Python >=3.9; 3.9.4 Requires-Python >=3.9; 4.0.0 Requires-Python >=3.9; 4.0.0.dev3 Requires-Python >=3.9; 4.0.1 Requires-Python >=3.9; 5.5.0 Requires-Python >=3.9; 6.2.0 Requires-Python >=3.9; 6.3.0 Requires-Python >=3.9; 6.3.1 Requires-Python >=3.9; 8.3.3 Requires-Python <3.13,>=3.9; 9.0.0 Requires-Python >=3.9; 9.1.0 Requires-Python >=3.9; 9.1.1 Requires-Python >=3.9
ERROR: Could not find a version that satisfies the requirement torch-sparse==0.6.18+pt20cu117 (from versions: 0.1.0, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.5.1, 0.6.0, 0.6.1, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.6.7, 0.6.8, 0.6.9, 0.6.10, 0.6.11, 0.6.12, 0.6.13, 0.6.14, 0.6.15, 0.6.16, 0.6.17, 0.6.18)
ERROR: No matching distribution found for torch-sparse==0.6.18+pt20cu117

failed

CondaEnvException: Pip failed

**Comment by huaishengzhu** [comment]
Thanks for your interest about our work and pointing out this issue! If you can't successfully install it through the environment.yml, you can download our package in the Google drive as mentioned as an alternative way in the Environment Preparation part.

Best,

**Comment by Sakura1a2a3a** [comment]
I tried to use the downloaded package for environment setting. But there is no rdkit package in it, specifically not version 2023.9.3(not compatible with python 3.8). If it is not, then in the denoising_diffusion.py it cannot import rdRascalMCES from rdkit.Chem. Could you please provide a solution?

**Comment by huaishengzhu** [comment]
Hi,

We have updated new version of our package by repacking it and we test it on another server which has the rdkit package. Can you try to redownload it and try it again?

Best,

**Comment by Sakura1a2a3a** [comment]
Yeah, I have reproduced your performance successfully. Thank you so much !
